IBPSA/BoundaryConditions/SolarGeometry/BaseClasses/SolarAzimuthWithEvent.mo

```diff
@@ -0,0 +1,31 @@
+within IBPSA.BoundaryConditions.SolarGeometry.BaseClasses;
+function SolarAzimuthWithEvent "Determines solar azimuth with event"
+  input Real solAziTem "Temporary solar azimuth";
+  input Real solTim "Solar time";
+  input Real day "Number of seconds in day";
+  output Real solAzi "Solar azimuth";
```

Comment:
Add units where applicable

Reply:
Addressed in https://github.com/ibpsa/modelica-ibpsa/pull/1374/commits/f5d035b7b096c40ac878c4d525be00309bce9265.